pkg/log/pipeline_reader.go

```diff
@@ -158,8 +158,12 @@ func (r *Reader) waitUntilAvailable(timeout time.Duration) error {
 			}
 		case <-time.After(timeout * time.Second):
 			watchRun.Stop()
+			if err = isPipelineRunRunning(run.Status.Conditions); err != nil {
+				fmt.Fprintln(r.stream.Err, err.Error())
+				return nil
```

Comment:
Per comment below on returning a bool, instead of printing to Stderr here, the message could just be written to Stdout.

Reply:
addressed